addons/web/static/src/core/notebook/notebook.scss

```
@@ -78,4 +77,12 @@
             }
         }
     }
+    @media only screen and (max-width: 767px) {
+        .o_notebook_content .oe-toolbar {
+            position: sticky;
+            top: 0;
+            margin: 0 var(--notebook-margin-x, -16px);
```

Since we're playing with horizontal  margins only here, is better to keep the vertical ones untouched.
One line more, but less headaches in case of future adaptations.

 So:
```suggestion
            margin-left: var(--notebook-margin-x, -16px);
            margin-right: var(--notebook-margin-x, -16px);
```

...btw,  why we need  `16px` ?